tox.ini

```diff
@@ -3,6 +3,9 @@ minversion = 3.3.0
 envlist = py311,py310,py39,py38,lint
 isolated_build = true
 
+[stestr]
+group_regex=([^\.]+\.)+
```

Comment:
I think this should go in `.stestr.conf`. The stestr docs say that the tox entry is ignored if `.stestr.conf` exists.

Reply:
Good catch, thanks. I moved it and changed the option to `parallel_class=True` which is easier to understand.